qiskit_machine_learning/kernels/quantum_kernel.py

```diff
@@ -90,6 +112,108 @@ def quantum_instance(
+            }
+
+        self._user_parameters = user_params
+
+    def assign_user_parameters(
+        self, values: Union[Mapping[Parameter, ParameterValueType], Sequence[ParameterValueType]]
+    ) -> None:
+        """
+        Assign user parameters in the QuantumKernel feature map.
+
+        Args:
+            values (dict or iterable):
+                Dict {parameter: value, ...} or {parameter: parameter}:
+                The keys of the parameter dictionary must be Parameter instances in the
+                feature map circuit. The values of the dictionary should be either numeric
+                values {param:float} or copies of the param key {param:param}. Passing a
+                copy of the param key as a value causes that parameter to be ignored.
+                This allows the circuit's user parameters to be bound consecutively.
+
+                Iterable [value_0, value_1, ..., value_N]:
+                If a list of real values is passed, the elements are assigned to the parameters
+                stored in the user_parameters field.
+
+        Return:
+            None
+
+        Raises:
+            ValueError: Incompatible number of user parameters and values
+        """
+        if not isinstance(values, dict):
+            if self._user_parameters is None:
+                raise ValueError(
+                    """
+                    An iterable of user parameter values was given, but no user
+                    parameters were defined for the QuantumKernel.
+                    """
+                )
+
+            if len(values) != len(self._user_parameters):
+                raise ValueError(
+                    f"""
+                    An iterable of user parameter values was given as input,
+                    but the number of parameter values ({len(values)}) does not match the
+                    number of user parameters tracked by the QuantumKernel
+                    ({len(self._user_parameters)}).
+                    """
+                )
+
+            param_binds = {param: values[i] for i, param in enumerate(self._user_parameters)}
+            values = param_binds
+
+        if self._user_param_binds is None:
+            self._user_param_binds = values
+        else:
+            self._user_param_binds.update(values)
+        self._feature_map = self._unbound_feature_map.assign_parameters(self._user_param_binds)
+
+    @property
+    def user_param_binds(self) -> Mapping[Parameter, float]:
+        """Return a copy of the current user parameter mappings for the feature map circuit."""
+        return copy.deepcopy(self._user_param_binds)
+
+    def bind_user_parameters(
+        self, values: Union[Mapping[Parameter, float], Sequence[float]]
+    ) -> None:
+        """
+        Alternate function signature for assign_user_parameters
+
+        Args:
+            values (dict or iterable): {parameter: value, ...} or [value1, value2, ...]
+
+        Return:
+            None
```

Comment:
Similarly here - I don't think anything needs to be said if it doesn't return anything